tools/arm64-build-wxwidgets.bat

```diff
@@ -0,0 +1,20 @@
+# taken from https://github.com/pit-ray/win-vind/issues/191#issuecomment-1671656805
+
+# Assume wxwidgets source already available. For fresh copy use
+#    .\tools\arm65-clean-checkout-wxwidgets.bat
+
+cd libs
+if %errorlevel% neq 0 exit /b %errorlevel%
+
+cd wxWidgets
+if %errorlevel% neq 0 exit /b %errorlevel%
+cd build/msw
+if %errorlevel% neq 0 exit /b %errorlevel%
+rem call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsamd64_arm64.bat"
+rem if %errorlevel% neq 0 exit /b %errorlevel%
+call "c:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools\VsDevCmd.bat"
```

Comment:
`VsDevCmd.bat` builds 32bit x86-native by default, so you should use `vsvarsamd64_arm64.bat` to build arm64 code.

Reply:
Yes, that's what those docs say, but it's not what is on my machine.

<details><summary>Listing</summary>

```powershell
C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools> ls


    Directory: C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        2023-04-21     13:27                1033
d-----        2023-04-21     13:26                devinit
d-----        2023-04-21     11:07                en
d-----        2023-04-21     11:07                vsdevcmd
-a----        2023-04-21     17:49          50592 errlook.exe
-a----        2023-04-21     17:49           7427 errlook.hlp
-a----        2023-04-21     17:49          36272 guidgen.exe
-a----        2023-04-21     17:48          20486 Launch-VsDevShell.ps1
-a----        2023-04-21     17:48            103 LaunchDevCmd.bat
-a----        2023-04-21     17:49          31152 makehm.exe
-a----        2023-04-21     17:48          35728 Microsoft.VisualStudio.DevShell.dll
-a----        2023-04-21     17:52         115559 spyxx.chm
-a----        2023-04-21     17:52         695208 spyxx.exe
-a----        2023-04-21     17:52         163200 spyxxhk.dll
-a----        2023-04-21     17:52         181136 spyxxhk_amd64.dll
-a----        2023-04-21     17:52         115559 spyxx_amd64.chm
-a----        2023-04-21     17:52         945080 spyxx_amd64.exe
-a----        2023-04-21     17:51         319384 vcconvertengine.dll
-a----        2023-04-21     17:51         369600 VCProjectConversion.dll
-a----        2023-04-21     17:48          11073 VsDevCmd.bat
-a----        2023-04-21     17:48            611 VsMSBuildCmd.bat


C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools>
```
</details>

**FWIW** since I'm no longer comfortable at a windows command line, I am doing this all from VS-Code. I'm running the batch files from a powershell-opened-in-vscode.

Reply:
I have tried BuildTools edition of Visual Studio, not Community.
There are these various command line tools in the `C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build`, but the arm64 tool does not exist.
Even worse, I cannot find the arm64 option for the BuildTools edition in the Visual Studio Installer.

However, if you install Visual Studio 2019 Comminy rather than BuildTools, these command files are available in the `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build`.
The arm compiler is not install as default, you should enable `MSVC v 142 - VS 2019 C++ ARM64 Build tool` of `Desktop development with C++` in Visual Studio Installer.